website/docs/r/resource_provider_registration.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+subcategory: "Base"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_resource_provider_registration"
+description: |-
+    Registers a resource provider.
+---
+
+# azurerm_resource_provider_registration
+
+Registers a resource provider.
```

Comment:
We might want to point out that this is only applicable right now when the provider option to skip registration is configured

Reply:
I`ll add that to the next iteration.